### PR TITLE
Add MuchRails::Action result files

### PR DESCRIPTION
**Pull request by nweathersbee**
This adds all result oriented files to `MuchRails::Action`. This
is part of an effort to bring in all view action support files.

### CODE REVIEW

File: lib/much-rails/action/base_command_result.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "much-rails/action/base_result"
+
+# MuchRails::Action::BaseCommandResult is a base result that, when
+# executed, runs a generic controller command with some given args.
+module MuchRails; end
+module MuchRails::Action; end
+class MuchRails::Action::BaseCommandResult < MuchRails::Action::BaseResult
+
```
Comment (kellyredding) [inline lib/much-rails/action/base_command_result.rb]: No empty line between class opening and `attr_reader`

File: lib/much-rails/action/render_result.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "much-rails/action/base_result"
+
+# MuchRails::Action::RenderResult is a result returned by calling a view
+# action that directs the controller to render a response.
+module MuchRails; end
+module MuchRails::Action; end
+class MuchRails::Action::RenderResult < MuchRails::Action::BaseResult
+
```
Comment (kellyredding) [inline lib/much-rails/action/render_result.rb]: no empty line here.

File: lib/much-rails/action/unprocessable_entity_result.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# calling a view action that is not valid. It returns JSON with the validation
+# details.
+#
+# This result is only returned if, after validating the action, there are
+# errors. Most commonly this occurs when validating form submission params.
+module MuchRails; end
+module MuchRails::Action; end
+class MuchRails::Action::UnprocessableEntityResult <
+        MuchRails::Action::BaseResult
+
```
Comment (kellyredding) [inline lib/much-rails/action/unprocessable_entity_result.rb]: why do you keep adding a blank line between class opening and `attr_reader`?

**Review by kellyredding — APPROVED** [review]
@nweathersbee a few minor comments but otherwise good.

File: lib/much-rails/action/unprocessable_entity_result.rb (hunk 3 above)
Comment (nweathersbee) [reply lib/much-rails/action/unprocessable_entity_result.rb]: Must have done it on the 1st one and followed the bad pattern, no real reason.

**Pull request merged**